server/mediaserver.go

```diff
@@ -466,10 +409,10 @@ func gotRTMPStreamHandler(s *LivepeerServer) func(url *url.URL, rtmpStrm stream.
 						}
 						cond.L.Unlock()
 
-						if !eth.VerifySig(rpcBcast.Job().TranscoderAddress, crypto.Keccak256(segHashes...), res.Sig) {
-							glog.Error("Sig check failed for segment ", seg.SeqNo)
-							return
-						}
+						// if !eth.VerifySig(transcoderAddress, crypto.Keccak256(segHashes...), res.Sig) { // ANGIE - GET TRANSCODER ADDRESS HERE
```

Comment:
Remove?

Reply:
We really should be verifying the segment sig once we have the transcoder address, so I wouldn't remove this part entirely.